pkg/accounting/accounting.go

```diff
@@ -0,0 +1,210 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package accounting
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/ethersphere/bee/pkg/logging"
+	"github.com/ethersphere/bee/pkg/p2p"
+	"github.com/ethersphere/bee/pkg/storage"
+	"github.com/ethersphere/bee/pkg/swarm"
+)
+
+var _ Interface = (*Accounting)(nil)
+
+// Interface is the main interface for Accounting
+type Interface interface {
```

Comment:
Because that is how it's done in some other packages as well (e.g. addressbook, retrieval, pullsync). E.g. if this were called `Accounting`, the main implementation would have to have another name. If used outside the package it will always be called `accounting.Interface` which makes clear what it is.